.gitignore

```diff
@@ -60,3 +60,52 @@ bin/
 *.class
 build/
 .gradle/
+
```

Comment:
These seem to be gitignore entries better suited for a personal `~/.gitignore` - do they need to be in the phantomjs repo?